src/application/selector/json/JsonResultConverter.ts

```diff
@@ -1,5 +1,5 @@
 class JsonResultConverter {
-  static toString(results: any) {
+  static toString(results: any[]) {
```

Comment:
try unknown[] instead of any[]

Reply:
Tried, didn't work. `any` is what corresponding lib (`jsonpath`) returns so I'd like to leave it as-is.